spec/factories.rb

```diff
@@ -1,2 +1,13 @@
-FactoryBot.define do  
+FactoryBot.define do
+  factory :family do
+    first_name "Ro"
+    last_name "Liso"
+    address "San Cesareo"
+    animals "No"
+    capability 3
+    prof false
+    preferred_gender "female"
+    bus_driver false
+    active true
```

Comment:
I would only keep the attributes that have to exist in order for the factory to be valid.
Also if a value is set by default, we don't need to manually set it here.

Reply:
Thanks Elle, I will delete all the attributes that are not necessary for validation.